Documentation/git-config.txt

```diff
@@ -235,8 +235,12 @@ FILES
 If not set explicitly with `--file`, there are four files where
 'git config' will search for configuration options:
 
+$PROGRAMDATA/Git/config::
+	System-wide configuration file for *all* Git-related software running
+	on the machine.	Typically `$PROGRAMDATA` points to `C:\ProgramData`.
+
 $(prefix)/etc/gitconfig::
-	System-wide configuration file.
+	System-wide configuration file for this installation of Git for Windows.
```

Comment:
As we want to stay as close to the Git project (i.e. the platform-independent upstream project), it should be a good idea to keep the original intact, and only add something like:

> (Windows-only) this file is *not* intended to be shared with other Git implementations (such as JGit, libgit2, etc).